src/helpers/settings.ts

```diff
@@ -21,6 +21,47 @@ abstract class BaseSettings {
     }
 }
 
+export interface IPinnedQuery {
```

Comment:
I don't think you need to export this interface (only references is in this particular file).